modules/vinichuk_t_binary_search/test/test_Vinichuk_Timofey_binary_search.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 Vinichuk Timofey
+
+#include <gtest/gtest.h>
+#include <numeric>
+#include "include/BinarySearch.h"
+
+
+TEST(BinarySearchTest, test_find_middle) {
+    std::vector<int> v = {1, 2, 3, 4, 5, 6, 7};
```

Comment:
```suggestion
    std::vector<int> v{1, 2, 3, 4, 5, 6, 7};
```
Также поправьте пожалуйста в других местах